content/docs/1.2.0/tutorial/introduction.md

```diff
@@ -0,0 +1,55 @@
+# Introduction
+
+In this tutorial, we'll cover how to create, deploy, and manage our first microservices using ioFog.
+
+<aside class="notifications note">
+  <h3><img src="/images/icos/ico-note.svg" alt=""> Familiar with the Core Concepts?</h3>
+  <p>If you aren't already familiar with the core concepts of ioFog, you'll want to check out <a href="../getting-started/core-concepts.html">our Core Concepts</a> section.</p>
+</aside>
+
+## Setup Demo Project
+
+In this tutorial, we are going to extend on the ioFog demo project we have previously used in [Quickstart](../getting-started/quick-start.html). Please follow the instructions on how to download all prerequisites required for running the demo project.
+
+We can skip this section if we have already downloaded the demo project from [Quickstart](../getting-started/quick-start.html).
```

Comment:
Do we use the same link twice here? Or we do we want to replace one of the two links?